### DISCUSSION TITLE
Who's using React Suite?

### DISCUSSION THREAD

**Discussion by SevenOutman** [rsuite/rsuite#11] [Show and tell]
If you/your company is using React Suite either publicly or internally, we would love to know about it 😍! 

Don't be shy to share with us either of your:
- 🏢 Company name
- 🖼️ Screenshots
- 🔗 Link to your website
- 🆒 Supportive/encouraging words

**Comment by windgaucho** [comment]
Hello, congratulations for this great suit of ui react components, it looks great !!
I wanted to know if there are some design or project examples from which I can start using rsuite.

**Reply by md1116** [reply]
Hi there,
I am looking for a react data table to use for a large data set that will be working with Tailwind CSS.
Is there away to use this library with tailwind/bootstrap?

thank you for your hard work.

**Comment by simonguo** [comment]
@windgaucho ,
https://github.com/rsuite/rsuite-management-system
You can keep an eye on this library, which is an example project that will be open source in the near future.

**Reply by RajeshReddy999** [reply]
Hey @windgaucho  
Thanks for the template 
Its pretty good for beginners to understand how to use rsuite

**Comment by windgaucho** [comment]
Thank you very much, I will start using this framework!! it is look awesome!!

**Comment by gcv5** [comment]
```js
import React, { Component } from 'react';
import { Navbar, Nav, Icon, Dropdown } from 'rsuite';
import 'rsuite/dist/styles/rsuite.min.css';

class Header extends Component {
  render() {
    return (
      <Navbar>
        <Navbar.Header>
          <a href="#" className="navbar-brand logo">RSUITE</a>
        </Navbar.Header>
        <Navbar.Body>
          <Nav>
            <Nav.Item icon={<Icon icon="home" />} >Home</Nav.Item>
            <Nav.Item>News</Nav.Item>
            <Nav.Item>Products</Nav.Item>
            <Dropdown title="About">
              <Dropdown.Item>Company</Dropdown.Item>
              <Dropdown.Item>Team</Dropdown.Item>
              <Dropdown.Item>Contact</Dropdown.Item>
            </Dropdown>
          </Nav>
          <Nav pullRight>
            <Nav.Item icon={<Icon icon="cog" />} >Settings</Nav.Item>
          </Nav>
        </Navbar.Body>
      </Navbar>
    );
  }
```
这样使用时，页面没有在示例页面中#body .navbar-brand的样式，请问这些样式是从哪加载的？

**Comment by simonguo** [comment]
@gdx945  https://github.com/rsuite/rsuite.github.io/blob/master/src/less/main.less#L715

**Comment by ghost** [comment]
Been using it in 4 Projects now and I love it. If you need any contributors dont bother to hit me up!

**Comment by Link-Kou** [comment]
我们公司目前就使用了此框架来做中后台。
![4.0.png](https://uploads.codesandbox.io/uploads/user/54d06f4d-17b5-465c-83df-cb94c7484cde/WgTH-2019-01-10_17-04-14.png)
![5.0.png](https://uploads.codesandbox.io/uploads/user/54d06f4d-17b5-465c-83df-cb94c7484cde/kuLF-2019-01-10_17-05-19.png)

**Comment by shiyachuang** [comment]
Thanks for developing this component is great, especially the virtual components.  👍👍👍

**Comment by benni3223** [comment]
just started my first application with this framework, i was searching a nice looking framework for b2b applications, without the playfulness like in material-ui as example. First i wanted to use semantic, but then i saw your framework i have to say, it's awesome. Really nice looking and easy to understand. I really love that Pickers with Searching etc, perfect for my usecase, and the combination of the ripple-buttons and normal input components is really good fitting. 
Really good work, will definitely recommend it!

**Comment by Ericnr** [comment]
Just found out about the lib and it seems amazing. Seriously considering using it. The only thing that threw me off a little was the fact there is no prop you can pass to disable the buttons ripple effect. Think I would have to roll out my own button component.

**Comment by simonguo** [comment]
@Ericnr 

Disable ripple effect：
```less
@import '~rsuite/src/styles/index';
@button-ripple: false;
```

A detailed description of the custom theme. https://rsuitejs.com/en/guide/themes

**Comment by Ericnr** [comment]
oke thats perfect thank you

**Comment by totoptech** [comment]
Hi everyone,

I have some problems in implementing rsuite table components.

![image](https://user-images.githubusercontent.com/57599786/73425661-95ddfa80-4332-11ea-94f7-4212c230d3d9.png)

As you can see, horizontal scroll bar always does not work properly at the first time.
If I resize browser several time, it does work well.

Please help me to handle this problem.

**Comment by VaibhavAcharya** [comment]
This framework is amazing. I have just discovered it and am considering to use it quite often in future projects. I just want to say thank you developer community.

**Comment by pabloiea1995** [comment]
Great work guys! This library makes it so easy and fast to develop a very well contructed and professional UI. I have used it for my first freelance project and the result has been amazing... Got the result o 100 working hours in just 30.
I have a few ideas to contribute, i wil try to develop them by my self and contribute!

**Comment by nimrod-cohen** [comment]
Thank you for this awesome library, great work, I'm using it for our backoffice and it saved me so much time! kudos!

**Comment by eDubrovsky** [comment]
`Filmustage` - is the first AI-based solution for automatically breakdown movie screenplays in seconds.

https://filmustage.com - Home
https://app.filmustage.com - App with `rsuite`

Thank you for the amazing and beautiful framework. It has everything we need and more :) Thank you also for the quick issue solving.

![](https://i.imgur.com/vsbYZut.png)
![](https://i.imgur.com/FBa30ll.png)
![](https://i.imgur.com/AKV0jiL.png)

**Comment by shiqiongzhen** [comment]
who know how to fix it? hurry！
![image](https://user-images.githubusercontent.com/30165186/110786833-8eb04c80-82a7-11eb-94ca-2b9e815ab2fe.png)

**Reply by shiqiongzhen** [reply]
i fixed it out when i update rsuite version 4.9.2 to 4.9.3 ! But i don't know why the version of icon package update could influent the old version of rsuite package !
and i want to descripte the question using Chinese, who 大佬 can pull me into the group！

**Comment by shiqiongzhen** [comment]
no one reply me in this code（cry for carrying me！）
![image](https://user-images.githubusercontent.com/30165186/110787384-375eac00-82a8-11eb-9036-164833bdda5b.png)

**Comment by BossBele** [comment]
:office: Mabrouks
:cool: RSuite has saved us a lot of time. Its design is exquisite. It is a milestone in what designers can achieve through component creation. Well done!

**Comment by preetkaran20** [comment]
- 🏢 Owasp (OpenSource)
- 🔗 https://github.com/SasanLabs/VulnerableApp-facade and https://github.com/SasanLabs/VulnerableApp
- 🆒 You guys have done an amazing job with a fantastic library as well as very detailed documentation. It has a very less learning curve and has really saved a lot of time in building the frontend.
Keep up the good work ❤️.

Initial project design:
![image](https://user-images.githubusercontent.com/13570884/122669516-6952a900-d1db-11eb-9b01-ff430c878947.png)

**Comment by maxmert** [comment]
🏢 Writex.io – online text editor with extreme possibilities.
🔗 [https://app.writex.io](https://app.writex.io)
🆒 Thanks for this project! Would **love** to collaborate to continue to improve it! Really enjoyed to use it ❤️

<img width="1359" alt="Screenshot 2021-06-21 at 17 46 14" src="https://user-images.githubusercontent.com/864232/122790751-ec95fc80-d2b8-11eb-83e8-93538d557380.png">
<img width="1199" alt="Screenshot 2021-06-21 at 17 45 14" src="https://user-images.githubusercontent.com/864232/122790767-f0298380-d2b8-11eb-927f-54380e943c62.png">
<img width="1359" alt="Screenshot 2021-06-21 at 17 46 07" src="https://user-images.githubusercontent.com/864232/122790785-f4ee3780-d2b8-11eb-937c-bb39349e6028.png">
<img width="1199" alt="Screenshot 2021-06-21 at 17 45 43" src="https://user-images.githubusercontent.com/864232/122790802-f7509180-d2b8-11eb-82ca-2274c8715244.png">

**Comment by 0xseven7** [comment]
NICE PROJECT!!! I'm going to consider replacing the heavy ANT DESIGN!

**Comment by gkirthiv** [comment]
Are you using bootstrap framework?
just wanted to know if I am able to use the bootstrap classes.

**Reply by SevenOutman** [reply]
Hi @gkirthiv, this thread turned out to be a collection of showcases. Please [open a new discussion](https://github.com/rsuite/rsuite/discussions) thread if you would like to ask a question.

**Comment by ObaPaguei** [comment]
Hey guys!

I have questions about the Uploader component.
How do I capture the file that was selected?
I dont have upload endpoint. I save the document in database.

**Reply by SevenOutman** [reply]
Hi @ObaPaguei, this discussion turned out to be a collection of showcases. I'm going to mark your comment as off-topic. Please [open a new discussion](https://github.com/rsuite/rsuite/discussions) thread if you would like to ask a question about using rsuite.

**Comment by guidone** [comment]
Hi, I love this UI lib, I think you might be interested in LetsForm, a tool to create forms using most famous UI libs, including RSuite

https://letsform.dev

and it comes with its one Designer to create forms

https://designer.letsform.dev

<img width="1468" alt="Screenshot 2023-06-15 at 11 04 23" src="https://github.com/rsuite/rsuite/assets/502519/d30838b2-eb0a-4b76-80ea-9b199b782313">

**Reply by SevenOutman** [reply]
This is cool!

**Reply by myrddral** [reply]
I've been looking for such a tool for a while but even the promising ones failed to impress in the end due to missing critical features or a ton of bugs.
**_But this one actually rocks, amazing work!_** Great selection of UI libs to bundle with my fav form management library. Honestly there is not much I can think of as missing.

**Reply by guidone** [reply]
Thanks! Would you like to help beta-testing?

**Reply by myrddral** [reply]
sure, glad to help :)

**Comment by shiqiongzhen** [comment]
小施已收到你的来信，祝你生活愉快~

**Comment by Azzmeth** [comment]
i need table  pagination table search ,sort ,table status filter

**Reply by SevenOutman** [reply]
Hi @Azzmeth, this discussion is meant for showcases. If you have feature requests or usage questions please open separate discussions or issues.

**Comment by shiqiongzhen** [comment]


**Reply by Azzmeth** [reply]
hii

**Comment by shiqiongzhen** [comment]
小施已收到你的来信，祝你生活愉快~

**Comment by phil-baller** [comment]
Really great UI library. Every component has perfectly been crafted to suit its purpose. 
This is definitely the library of the future together with shadcn-ui

I worked with the tables and created a custom pagination component to handle server side pagination and it rocked. Glad i fell on this library. Can't wait to test out all the other components in my upcoming project. Let's see how far it goes

**Comment by shiqiongzhen** [comment]
小施已收到你的来信，祝你生活愉快~

**Comment by Alexandro-Bolfa-se** [comment]
I am using Rsuite of a date picker and I am also using the locale.
I have noticed that both files have the same **code: "es-AR"**. Shouldn't these be different?
node_modules\rsuite\esm\locales\es_AR.js and node_modules\rsuite\esm\locales\es_ES.js
![Screenshot 2025-05-20 095254](https://github.com/user-attachments/assets/21874835-ac59-4def-aac0-9a6b38948c92)
![Screenshot 2025-05-20 095346](https://github.com/user-attachments/assets/fcaa95ed-802b-4231-91f3-55173c1658c4)

**Reply by simonguo** [reply]
Fixed in https://github.com/rsuite/rsuite/pull/4293

**Comment by shiqiongzhen** [comment]
小施已收到你的来信，祝你生活愉快~

**Comment by rainy14f** [comment]
This is a very good UI framework, simple, clean yet comprehensive. The functionalities are well thought-through. Well done! 
I've been using it in my work (I can't mention the company name though).

**Reply by rainy14f** [reply]
Forgot to mention, I am using it in a SaaS product which is planned to open for business in the upcoming months.

**Comment by shiqiongzhen** [comment]
小施已收到你的来信，祝你生活愉快~
